### ISSUE TITLE
Extra line break in output

### ISSUE THREAD

**Issue by jslaterTO** [bdkjones/CodeKit2#239]
In b14 and b15, it seems that with any request CodeKit serves with its server (at least if it's using an external server - haven't tried without), it seems to be inserting a an additional line break at the start of requests.  In particular, this is an issue with requests expecting JSON, since that makes the syntax invalid.  As far as I can remember, this didn't happen prior to b14.

This is happening on 10.9.1, with MAMP Pro 2.2.  Let me know if you'd like any further details - thanks in advance!

**Comment by bdkjones** [comment]
I assume you mean the response, not request?

And is the linebreak at the very start, before the headers? Please post an exact copy of the response you're seeing along with the request that was sent.

**Comment by jslaterTO** [comment]
Yeah sorry, I meant response.  On further inspection, it looks like the end of the response is getting truncated as well - all HTML and JSON responses are also losing the last two characters (though others like CSS, JS, and images don't seem to be affected).  Also, I tried out a dummy project with a couple static files, turned off the external server, and this didn't occur.

Here's what I'm getting via CodeKit's server: http://c.jslater.ca/Tn9Y

The correct response should be: `{"success":true}` (which I get accessing the site directly and submitting the form in question)

One thought I had was disabling mod_deflate, which didn't change anything.  Also, it's worth noting that the same thing occurs in Chrome.

Also, for shits and giggles I decided to try pointing the external server address to random websites to see what would happen - this bug occurred with Google, Twitter and Facebook's home pages, but not Apple's.
